frontend/src/lib-customizations/espoo/citizen/assets/i18n/sv.tsx

```diff
@@ -149,40 +149,40 @@ const sv: Translations = {
     backToSearch: 'Tillbaka till sökning'
   },
   calendar: {
-    title: 'Kalenteri',
-    holiday: 'Pyhäpäivä',
-    absent: 'Poissa',
+    title: 'Kalender',
+    holiday: 'Helgdag',
+    absent: 'Frånvarande',
     absences: {
-      SICKLEAVE: 'Sairauspoissaolo',
-      PLANNED_ABSENCE: 'Vapaapäivä'
+      SICKLEAVE: 'Sjukskriven',
+      PLANNED_ABSENCE: 'Ledig dag'
     },
-    newReservationOrAbsence: 'Varaus / Poissaolo',
-    newAbsence: 'Ilmoita poissaolo',
-    newReservationBtn: 'Tee varaus',
-    noReservation: 'Ei varausta',
+    newReservationOrAbsence: 'Reservation / Frånvaro',
+    newAbsence: 'Anmäl frånvaro',
+    newReservationBtn: 'Reservera',
+    noReservation: 'Ingen reservation',
     reservationModal: {
-      title: 'Tee varaus',
-      selectChildren: 'Valitse lapset',
-      dateRange: 'Varauksen voimassaolo',
-      dateRangeLabel: 'Tee varaus päiville',
-      times: 'Kellonaika',
-      businessDays: 'Ma-Pe',
-      postError: 'Varauksen luominen ei onnistunut',
-      repeats: 'Toistuu',
+      title: 'Reservera',
+      selectChildren: 'Utvalda barn',
+      dateRange: 'Reservationens giltighet ',
+      dateRangeLabel: 'Reservera dagarna',
+      times: 'Tid',
+      businessDays: 'Mån-Fre',
+      postError: 'Misslyckades med att reservera',
+      repeats: 'Upprepa',
```

Comment:
Maybe even upprepning?

Reply:
Or upprepas. Probably not upprepar though. At least that's my feeling 😅

Reply:
I think it should either be:
* impreative (upprepa) 
* passive (upprepas)
* noun (upprepning)
I don't think present (upprepar) works since there is no subject.